README.md

```diff
@@ -1 +1,175 @@
+
+> Describe between 1-3 key features of the application.
+
+- **[Undestand how I can build a website by myself from scratch]**
+- **[Give myself a challenge to test my knowledges]**
+- **[Get real Experiance]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+<!-- This website has not been deployed -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install Git on your computer following the steps [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- Install node js following the steps [here](https://www.pluralsight.com/guides/getting-started-with-nodejs)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  - cd my-folder
+  - git clone https://github.com/Kaghenimbale/Awesome-book.git
+
+### Install
+
+Install this project with:
+
+  - cd my-project
+  - gem install
+
+### Usage
+
+To run the project, execute the following command:
+
+  - cd my-project
+  - Open the index.html in the browser
+
+### Run tests
+
+To run tests, run the following command:
+
+  - cd my-project
+  - npx hint .
+  - npx stylelint "**/*.{css,scss}"
+
+### Deployment
+
+You can deploy this project using:
+
+This project has not been deployed
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Kagheni Mbale Christian**
+
+- GitHub: [@githubhandle](https://github.com/Kaghenimbale)
+- Twitter: [@twitterhandle](https://twitter.com/MbaleKagheni)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kagheni-mbale-401b90240/)
+
+👤 **Ernest Musong**
+
+- GitHub: [@githubhandle](https://github.com/ernestmusong)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Styling of the website]**
+- [ ] **[Add more logic]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a start ⭐️ If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks to cindy the Dsigner of the project Template
```

Comment:
- [ ] kindly check the acknowledgement as this is not a design template given to us by Cindy shin